src/styles/grade.scss

```diff
@@ -0,0 +1,26 @@
+.grade {
+  display: flex;
+  flex-direction: row;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), if properties are related to the flex layout, they should be grouped together and separated from other groups by an empty line. There should be an empty line before `flex-direction`.